compiler/ballerina-lang/src/main/java/io/ballerina/projects/util/ProjectConstants.java

```diff
@@ -29,6 +29,7 @@
 
     public static final String BALLERINA_TOML = "Ballerina.toml";
     public static final String DEPENDENCIES_TOML = "Dependencies.toml";
+    public static final String COMPILER_PLUGIN_TOML = "Compiler-plugin.toml";
```

Comment:
Should this be CompilerPlugin.toml ?

Reply:
I don't have a strong opinion on this. +1 if this is the pattern that we've used in other places